test/unittests/capi_test.cpp

```diff
@@ -12,3 +16,224 @@ TEST(capi, validate)
     wasm_prefix[7] = 1;
     EXPECT_FALSE(fizzy_validate(wasm_prefix, sizeof(wasm_prefix)));
 }
+
+TEST(capi, parse)
+{
+    uint8_t wasm_prefix[]{0x00, 0x61, 0x73, 0x6d, 0x01, 0x00, 0x00, 0x00};
+    auto module = fizzy_parse(wasm_prefix, sizeof(wasm_prefix));
```

Comment:
```suggestion
    auto module = fizzy_parse(wasm_prefix, std::size(wasm_prefix));
```

Reply:
Why is this better?

Reply:
Counts number of elements, not number of bytes. Here it is the same, so it does not matter.